README.md

```diff
@@ -64,6 +64,18 @@ With the following processes running as cron jobs on daily basis:
 
 On heroku these are configured using the scheduler.
 
+Monitoring
+----------
+This server will print log messages it is recommend run with `DEBUG` as
+`"* -superagent -6to5 -mocha:* -express:*"`.
+Notice that messages printed containing the substring `[alert-operator]` should
```

Comment:
Yeah, we should do something in taskcluster-base that wraps this with `console.error("[alert-operator]" + msg, args)`.
This way we can avoid typos and we can't hide them with `DEBUG` env var...

Filed bug 1144521 to discuss/implement a proper method for this...